### ISSUE TITLE
Editing in an Office 365 app

### ISSUE THREAD

**Issue by fredericklin** [MicrosoftDocs/OfficeDocs-SharePoint#497]
Clarify that you can still edit in the Online version of the Office 365 app where it states:

Ad-hoc external recipient:
Can edit in Word, Excel, PowerPoint, or other Office 365 apps - No

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f75d2b5a-bc14-e0f3-1269-716383939212
* Version Independent ID: 879c8636-ab9a-0440-039a-f3eac8d41a1f
* Content: [Secure external sharing in SharePoint Online](https://docs.microsoft.com/en-us/sharepoint/what-s-new-in-sharing-in-targeted-release)
* Content Source: [SharePoint/SharePointOnline/what-s-new-in-sharing-in-targeted-release.md](https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/blob/live/SharePoint/SharePointOnline/what-s-new-in-sharing-in-targeted-release.md)
* Service: **sharepoint-online**
* GitHub Login: @MikePlumleyMSFT
* Microsoft Alias: **mikeplum**

**Comment by nenonix** [comment]
Thank you for taking time to contribute clarity to the content. Am I right in assuming that you are suggesting that we differentiate between Office 365 apps and Office 365 Online in terms of access for ad-hock external recipients or do you think a note could be added to make this clear?

**Comment by fredericklin** [comment]
I actually assumed the "Office 365 app" is inclusive of the online browser interface which I discovered to be editable.  I think I misinterpreted it in retrospect.  There has also been some commentary from others (ie. blog comments) about not being able to open a Word/Excel/etc document in the rich client.  So perhaps to avoid confusion, clarification on the terminology would be good to mention what is possible with all of the different clients involved (mobile app, online, and rich client).
